setup.cfg

```diff
@@ -113,3 +113,11 @@ exclude =
 # This will be used when updating. Do not change!
 version = 3.2.3
 package = sympc
+
+[isort]
+multi_line_output = 3
```

Comment:
NITPICK: Shouldn't we also have ```profile=black``` ? (I am not sure if this is selecting black as the profiler)

Also, maybe it is worth to have this:
```
sections=FUTURE,STDLIB,THIRDPARTY,SYFT,LOCALFOLDER
```
I think this ^^ will put an order on importing stuff?

Also, they have this part.

```
import_heading_future=future
  import_heading_stdlib=stdlib
  import_heading_thirdparty=third party
  import_heading_syft=syft absolute
  import_heading_localfolder=syft relative
```
^^ I am not 100% sure if one depends on the other - but I think it would be nice to have an order on the order of imported libs :D

Reply:
As far as I understand... We don't have to have the profile **also** (see [this](https://pycqa.github.io/isort/docs/configuration/profiles/)). It's just another way of adding the same thing. But I set the configuration explicitly to be able to tune it as we want. 

Regarding the order.... That's a good point. I'll make the change so as to roughly follow that order too.

Reply:
I've changed the setup so as to have pretty much the same configuration as syft.